src/epcpm/cli/main.py

```diff
@@ -144,12 +150,15 @@ def docs(
     paths = epcpm.importexportdialog.paths_from_directory(target_path)
 
     loaded_project = epcpm.project.loadp(project)
-
+    product_specific_defaults_list = []
+    if product_specific_defaults:
+        product_specific_defaults_list = product_specific_defaults.split(",")
```

Comment:
Need to remove the leading and trailing spaces which users might enter.
```suggestion
        product_specific_defaults_list = product_specific_defaults.split(",")
        product_specific_defaults_list = [x.strip() for x in product_specific_defaults_list]
```

Reply:
[d408edf](https://github.com/epcpower/pm/pull/91/commits/d408edfe144b212c5b48952e5ec913cad3155485)